apps/central-scan/frontend/src/navigation_screen.tsx

```diff
@@ -69,6 +70,32 @@ const CentralScanAppLogo = styled(AppLogo)`
   }
 `;
 
+const SYSTEM_ADMIN_NAV_ITEMS = [
```

Comment:
I followed the pattern from VXAdmin [here](https://github.com/votingworks/vxsuite/blob/8c6e3ec3d6e46724b6f0d4985dbad719ee15c6bf/apps/admin/frontend/src/components/navigation_screen.tsx#L37) in CentralScan for defining the nav items. I found it much easier to reason about as a new developer, though I did slightly simplify it since the nav items are fewer.